tests/extension/oneapi_kernel_compiler_spirv/kernel_compiler_spirv.cpp

```diff
@@ -22,22 +22,26 @@
 
 namespace kernel_compiler_spirv::tests {
 
+static const std::vector<uint8_t> kernels{
```

Comment:
Add a comment explaining why it is not possible to have `std::vector<std::byte>` everywhere at the first place or use `std::vector<std::byte>`.